WebRoot/js/zimbraMail/share/view/dialog/ZmUploadDialog.js

```diff
@@ -407,29 +407,27 @@ ZmUploadDialog.prototype._addFileInputRow = function(oneInputOnly) {
 	var inputId = id + "_input";
 	var removeId = id + "_remove";
 	var addId = id + "_add";
-    var sizeId = id + "_size";
+	var detailId = id + "_detail";
 
 	var table = this._tableEl;
 	var row = table.insertRow(-1);
-
-    var cellLabel = row.insertCell(-1);
-    cellLabel.innerHTML = ZmMsg.fileLabel;
-
 	var cell = row.insertCell(-1);
 	// bug:53841 allow only one file upload when oneInputOnly is set
 	cell.innerHTML = [
-		"<input id='",inputId,"' type='file' name='",ZmUploadDialog.UPLOAD_FIELD_NAME,"' size=30 ",(this._supportsHTML5 ? (oneInputOnly ? "" : "multiple") : ""),">"
+		"<input id='",inputId,"' type='file' name='",ZmUploadDialog.UPLOAD_FIELD_NAME,"' size=30 ",(this._supportsHTML5 ? (oneInputOnly ? "" : "multiple") : ""),
+		" style='width: 0.1px;height: 0.1px;opacity: 0;overflow: hidden;position: absolute;z-index: -1;'>",
```

Comment:
intentionally kept here just because we are keeping the visibility 0 for this element and showing the label element (styling via css). 
We don't intend to allow input styling customisable or overridden.